tests/test_truncate.py

```diff
@@ -18,15 +18,17 @@ def test_interactive_truncate(self):
             fixedpoint_config=fixed100,
         )
 
-        with tfe.Session() as sess:
+        # TODO[Morten] remove this condition
+        if prot.tensor_factory not in [tfe.tensor.int64.int64factory]:
```

Comment:
I assume this is because interactive truncation doesn't work with int64. Maybe we should open an issue?